backend/controller/sql/queries.sql

```diff
@@ -160,7 +160,7 @@ SELECT d.min_replicas,
        r.endpoint,
        r.labels AS runner_labels
 FROM deployments d
-         LEFT JOIN runners r on d.id = r.deployment_id
+         LEFT JOIN runners r on d.id = r.deployment_id AND r.state = 'assigned'
```

Comment:
Hmmm actually if we just want to ignore dead runners, this probably isn't the right solution. We probably do want `!= 'dead'` so that it will also pick up runners that are reserved.

Reply:
I'm not 100% sure I'm right there, so it would be worth verifying if that does what we want...

Reply:
Makes sense, will fix